indico/util/string.py

```diff
@@ -612,10 +612,11 @@ def sanitize_css(self, style):
         return super().sanitize_css(style)
 
 
-def sanitize_html(string):
+def sanitize_html(string, *, allow_cid=False):
     css_sanitizer = IndicoCSSSanitizer(allowed_css_properties=BLEACH_ALLOWED_STYLES_HTML)
+    protocols = bleach.ALLOWED_PROTOCOLS | {'cid' if allow_cid else ''}
```

Comment:
```suggestion
    protocols = set(bleach.ALLOWED_PROTOCOLS)
    if allow_cid:
        protocols.add('cid')
```

Right now you add an empty string to the set...